pkg/registry/apis/provisioning/resources/replicator.go

```diff
@@ -53,6 +53,7 @@ func NewReplicator(
 
 // Sync replicates all files in the repository.
 func (r *Replicator) Sync(ctx context.Context) error {
+	// FIXME: how to handle the scenario in which folder changes?
```

Comment:
🤔  maybe our sync status can keep a hash of the key settings it is running against (eg, folder, id strategy, filters, ???) and if the hash changes, would need to wipe and recreate everything

Reply:
maybe that or the folder name in which the last successful sync happened. I will create a separate task for that because it isn't something introduced here.